docs/VnVPlan/VnVPlan.tex

```diff
@@ -188,44 +189,254 @@ \subsection{Tests for Functional Requirements}
 \wss{Include a blurb here to explain why the subsections below
 	cover the requirements.  References to the SRS would be good here.}
 
-\subsubsection{Area of Testing1}
-
-\wss{It would be nice to have a blurb here to explain why the subsections below
-	cover the requirements.  References to the SRS would be good here.  If a section
-	covers tests for input constraints, you should reference the data constraints
-	table in the SRS.}
+\subsubsection{Quotes}
 
-\paragraph{Title for Test}
+This section will contains tests covering the ``Quotes'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
 
 \begin{enumerate}
 
-	\item{test-id1\\}
+	\item \textbf{FRT-BE8-1}
 
-	Control: Manual versus Automatic
+	      Control: Manual
 
-	Initial State:
+	      Initial State: Customer Account with no quotes having been made by the account
 
-	Input:
+	      Input: Request past quotes
 
-	Output: \wss{The expected result for the given inputs}
+	      Output: A message saying there are no past quotes
 
-	Test Case Derivation: \wss{Justify the expected value given in the Output field}
+	      Test Case Derivation: There are no past quotes to display
 
-	How test will be performed:
+	      How test will be performed: The tester will request for past quotes through the user interface
 
-	\item{test-id2\\}
+	\item \textbf{FRT-BE8-2}
 
-	Control: Manual versus Automatic
+	      Control: Manual
 
-	Initial State:
+	      Initial State: Customer Account with completed quotes having been made by the account
 
-	Input:
+	      Input: Request past quotes
 
-	Output: \wss{The expected result for the given inputs}
+	      Output: A table with all the quotes
 
-	Test Case Derivation: \wss{Justify the expected value given in the Output field}
+	      Test Case Derivation: There are past quotes to display
 
-	How test will be performed:
+	      How test will be performed: The tester will request for past quotes through the user interface
+
+	\item \textbf{FRT-BE8-3}
+
+	      Control: Manual
+
+	      Initial State: Shop Owner/Employee Account with no quotes having been made against the shop
+
+	      Input: Request past quotes with quote ID, customer phone number, and/or customer name
+
+	      Output: A message saying there are no past quotes
+
+	      Test Case Derivation: There are no past quotes to display matching the inputted criteria
+
+	      How test will be performed: The tester will request for past quotes through the user interface
+
+	\item \textbf{FRT-BE8-4}
+
+	      Control: Manual
+
+	      Initial State: Shop Owner/Employee Account with completed quotes having been made against the shop
```

Comment:
maybe "existing quote request", "closed"/"open" quotes for terminology -- same for below? what is a completed quote?